lib/collector/serverless.js

```diff
@@ -35,6 +35,10 @@ class ServerlessCollector {
       agent_version: agent.version
     }
     this.payload = {}
+    this.pipePath = (agent.pipePath || process.env.pipePath) ?
```

Comment:
Should we use a better named env variable here like NEWRELIC_PIPE_PATH?

Reply:
I'd be in favor of that.